cpg_workflows/stages/happy_validation.py

```diff
@@ -4,75 +4,109 @@
 Content relating to the hap.py validation process
 """
 
-import logging
-
-from cpg_utils.config import get_config
+from cpg_utils import to_path
+from cpg_utils.config import config_retrieve
 from cpg_utils.hail_batch import get_batch
-from cpg_workflows.jobs.validation import parse_and_post_results, run_happy_on_vcf, validation_mt_to_vcf_job
+from cpg_workflows.jobs.validation import run_happy_on_vcf
+from cpg_workflows.stages.talos import query_for_latest_mt
 from cpg_workflows.targets import SequencingGroup
 from cpg_workflows.workflow import SequencingGroupStage, StageInput, StageOutput, get_workflow, stage
 
 
-@stage(analysis_type='custom', analysis_keys=['vcf'])
+@stage()
 class ValidationMtToVcf(SequencingGroupStage):
     def expected_outputs(self, sequencing_group: SequencingGroup):
-        return {
-            'vcf': (
-                sequencing_group.dataset.prefix()
-                / 'validation'
-                / get_workflow().output_version
-                / f'{sequencing_group.id}.vcf.bgz'
-            ),
-            'index': (
```

Comment:
is it ok to be removing the index output?

Reply:
Yeah I think so. The index is still written, but the index entry here is never used. The only useful entries in this output section are things to be used in subsequent stages, and the location of the index is always implicit from the location of the VCF